scripts/onboard_list_metadata.py

```
@@ -0,0 +1,54 @@
+import sys
+import urllib.request
+
+sys.path.insert(0, str(Path(__file__).resolve().parent))
+from _onboard_cli import fallback_api_key_from_env_files
+
+
+def _request_json(base_url: str, api_key: str, path: str) -> list[dict]:
+    req = urllib.request.Request(
+        f"{base_url.rstrip('/')}{path}",
+        headers={"X-OB-Api": api_key, "Content-Type": "application/json"},
+    )
```

Ruff (0.15.12)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)